### ISSUE TITLE
VRAM requirements for training

### ISSUE THREAD

**Issue by thanhnguyentung95** [2U1/Molmo-Finetune#5]
Thanks for sharing your work. Can you provide the VRAM requirements for each type of training (full, vision only, LLM only)?

**Comment by 2U1** [comment]
It doesn't support gradient checkpointing and other techniques so it takes more than it takes more than 96 GB. So I couldn't really test it for now.

**Comment by grzegorzj** [comment]
2x 80GB as far as I've observed, anything less, not so much.
